content/en/apps/tutorials/application-settings.md

```diff
@@ -0,0 +1,184 @@
+
+```json
+  "contact_types": [
+    {
+      "id": "district_hospital",
+      "name_key": "contact.type.district_hospital",
+      "group_key": "contact.type.district_hospital.plural",
+      "create_key": "contact.type.district_hospital.new",
+      "edit_key": "contact.type.place.edit",
+      "icon": "medic-district-hospital",
+      "create_form": "form:contact:district_hospital:create",
+      "edit_form": "form:contact:district_hospital:edit"
+    },
+    {
+      "id": "health_center",
+      "name_key": "contact.type.health_center",
+      "group_key": "contact.type.health_center.plural",
+      "create_key": "contact.type.health_center.new",
+      "edit_key": "contact.type.place.edit",
+      "parents": [
+        "district_hospital"
+      ],
+      "icon": "medic-health-center",
+      "create_form": "form:contact:health_center:create",
+      "edit_form": "form:contact:health_center:edit"
+    },
+    {
+      "id": "clinic",
+      "name_key": "contact.type.clinic",
+      "group_key": "contact.type.clinic.plural",
+      "create_key": "contact.type.clinic.new",
+      "edit_key": "contact.type.place.edit",
+      "parents": [
+        "health_center"
+      ],
+      "icon": "medic-clinic",
+      "create_form": "form:contact:clinic:create",
+      "edit_form": "form:contact:clinic:edit",
+      "count_visits": true
+    },
+    {
+      "id": "person",
+      "name_key": "contact.type.person",
+      "group_key": "contact.type.person.plural",
+      "create_key": "contact.type.person.new",
+      "edit_key": "contact.type.person.edit",
+      "primary_contact_key": "clinic.field.contact",
+      "parents": [
+        "district_hospital",
+        "health_center",
+        "clinic"
+      ],
+      "icon": "medic-person",
+      "create_form": "form:contact:person:create",
+      "edit_form": "form:contact:person:edit",
+      "person": true
+    }
+  ]
+```
+
+### 4. Set Replication Depth
+
+To configure replication depth for a specific role, edit the object corresponding to the `"replication_depth"` key in `app_settings.json`.
+
+```json
+{
+  "replication_depth": [
+    { "role": "district_manager", "depth": 1 },
+    { "role": "national_manager", "depth": 2 }
+  ]
+}
+```
+
+Configure the CHW role's depth to 2 by adding the following key/value pairs to the array above.
+
+```json
+{ "role": "chw", "depth": 2 }
+```
+
+### 5. Upload App Settings
+
+To upload app settings to your local instance, run the following command:
+
+```zsh
+medic-conf --url=https://<username>:<password>@localhost --accept-self-signed-certs upload-app-settings
+```
+
+Be sure to replace the values `<username>` and `<password>` with the actual username and password of your test instance.
```

Comment:
Using a "note" format here may make this less likely to be missed:
https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#notes-and-tips

Reply:
Updated.